book/types_of_data.md

```diff
@@ -34,89 +32,31 @@ You can cast a string into an Decimal with the `into decimal` command
 
 ## Strings
 
```

Comment:
I've made the string section more compact, and linked to the section "Working with strings", which introduces strings better (for example, it explains that the important property of double strings is that they treat escaped characters differently)